client/src/components/GirderFileManager.vue

```diff
@@ -0,0 +1,220 @@
+<!-- This component extends the FileManager component in order to provide a
+search bar and to collect and filter parameter options. -->
+
+<script>
+import { FileManager as GirderFileManager } from '@girder/components/src/components/Snippet';
+import GirderDataBrowser from './GirderDataBrowser.vue';
+import {
+  Breadcrumb as GirderBreadcrumb,
+} from '@girder/components/src/components';
+export default {
+  components: {
+    GirderBreadcrumb,
+    GirderDataBrowser,
+  },
+
+  mixins: [GirderFileManager],
+
+  inject: ['girderRest'],
+
+  data() {
+    return {
+      query: null,
+      allItems: [],
+      filteredItems: [],
+      input: '',
+      showPartials: false,
+      root: null,
+      outsideOfRoot: false,
+    };
+  },
+
+  computed: {
+    queryValues: {
+      get () {
+        if (this.query && !this.showPartials) {
+          return [this.query.value];
+        }
+        else if (this.showPartials) {
+          let values = this.filteredItems
+            .filter(item => {
+              return item.value.name.includes(this.input ? this.input : '');
+            })
+            .map(item => { return {...item.value, 'name': item.text}; });
+            if (!this.input)
+              this.$refs.query.blur();
+            return values;
+          }
+        return [];
+      }
+    },
+  },
+
+  async created() {
+    await this.setCurrentPath();
+    await this.getAllResults(this.location._id);
+    this.filteredItems = this.allItems;
+  },
+
+  watch: {
+    async query() {
+      if (this.input === '' || this.input && !typeof(this.query) == 'object'){
```

Comment:
This is not something for this PR. However, we might want to introduce lodash, to clean up some of these type comparisons.